src/tools/r2rdump/R2RDump.cs

```
@@ -144,6 +149,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 _options.Unwind = true;
                 _options.GC = true;
                 _options.SectionContents = true;
+                _options.EntryPoints = true;
```

Should we throw an error if the user specifies 'naked' AND 'raw'? They seem to contradict each other.

---

Fixed in 2nd commit.